checks.d/kubernetes.py

```diff
@@ -326,22 +326,25 @@ def _update_pods_metrics(self, instance, pods):
             "ReplicaSet",
         ]
 
-        controllers_map = defaultdict(int)
+        controllers_map = defaultdict(lambda: {'count': 0, 'namespace': None})
```

Comment:
This is a weird datastructure you use.

You should keep a `defaultdict(int)` (or a `collections.Counter`) where the keys would be a tuple `(namespace, controller)`